nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core/Utilities/OpenTelemetryLogger.cs

```diff
@@ -0,0 +1,55 @@
+using System.Globalization;
+
+using Microsoft.Extensions.Logging;
+
+using OpenTelemetry;
+using OpenTelemetry.Logs;
+
+namespace NuGetUpdater.Core
+{
+    public class OpenTelemetryLogger : ILogger, IDisposable
+    {
+        private readonly ILoggerFactory _loggerFactory;
+        private readonly Microsoft.Extensions.Logging.ILogger _logger;
+
+        public OpenTelemetryLogger()
+        {
+            _loggerFactory = LoggerFactory.Create(builder =>
+            {
+                builder.AddOpenTelemetry(logging =>
+                {
+                    logging.AddProcessor(new SimpleLogRecordExportProcessor(new CustomConsoleExporter()));
+                    logging.AddOtlpExporter();
+                });
+            });
+
+            _logger = _loggerFactory.CreateLogger<OpenTelemetryLogger>();
+        }
+
+        public void LogRaw(string message)
+        {
+            _logger.LogInformation(message);
+        }
+
+        public void Dispose()
+        {
+            _loggerFactory?.Dispose();
+        }
+    }
+
+    // We do this because the exporter that comes from AddConsoleExporter() prepends "LogRecord.Timestamp" in front of strings 
```

Comment:
Just to clarify, this is to avoid getting double timestamps?

Reply:
Well when I tested, LogRaw was basically coming out as:

"LogRecord.Timestamp:                     YYYY/MM/DD HH:MM:SS"

with the default console exporter.

So that's why I need the custom stuff that just spits out the message starting with YYYY/MM/DD HH:MM:SS .